pyproject.toml

```
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fool-ai-detector"
+version = "0.0.1"
+authors = [
+    { name="Raphael Weber"},
+    { name="Max Buchholz"},
+    { name="Lukas Müller"},
+    { name="Levi Otterbach"},
+]
+description = "Fooling ai detectors"
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "typer"
```

Could we make it install typer[all], so the output is pretty :)
```suggestion
    "typer[all]"
```